### DISCUSSION TITLE
Immich's timeline is wrong?

### DISCUSSION THREAD

**Discussion by sammyke007** [immich-app/immich#13222] [Q&A] [answered]
### The bug

Hi

Is it possible that the timeline in Immich is wrong?

Google Photo's timeline: shows eg. every day/part of the timeline in 2021 ABOVE the 2021 mark (IMHO the right way because 2021 is the mark for everything SINCE 2021).

Immich's timeline: shows eg. every day/part of the timeline in 2021 BELOW the 2021 mark.

I've attached two comparisons:

Google Photos - 07/2018
![ag-jul18](https://github.com/user-attachments/assets/d0ef3ec1-e29a-4dd1-9a63-2d472c0b4188)

Immich - 07/2018
![a-jul18](https://github.com/user-attachments/assets/934512b3-faca-47fe-956f-51f91cb5dea8)

or

Google Photos - 07/2022
![ag-jul22](https://github.com/user-attachments/assets/bfcfeff0-9e52-4cd4-aae9-ec065d84cabe)

Immich - 07/2022
![a-jul22](https://github.com/user-attachments/assets/a49a7261-5f42-45ab-ba4f-db9ff608e43e)

What do you think?

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.117.0

### Version of Immich Mobile App

v1.117.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Not required
```

### Your .env content

```Shell
Not required
```

### Reproduction steps

Not required

### Relevant log output

```shell
Not required
```

### Additional information

Country: Belgium
Timezone: Europe/Brussels

**Comment by bo0tzz** [comment]
It's just different

**Reply by sammyke007** [reply]
Yes indeed and I'm not complaining at all, but isn't Google Photo's way of labeling the year more logical?

**Reply by alextran1502** [reply, marked as answer]
I guess it comes down to the matter of preference. Have label similar to GPhotos previously. But then the current year on the top section won't be showing up since the year hasn't finished. So we opt-in to show it differently

**Reply by sammyke007** [reply]
Alright, tnx!
